mariadb_skysql/manifest.json

```diff
@@ -0,0 +1,58 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "bed39b01-bbc5-4819-9744-7cf4a3d90f6d",
+  "app_id": "mariadb-skysql",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor SkySQL MariaDB instances with DataDog.",
+    "title": "MariaDB SkySQL",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "image caption",
```

Comment:
The image should have a relevant caption

Reply:
Done